### DISCUSSION TITLE
Genera imágenes adaptables con la etiqueta picture

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#22] [Announcements]
# Genera imágenes adaptables con la etiqueta picture

Función para manipular imágenes con la etiqueta picture, para lograr mejor optimización en móviles.

https://www.zkreations.com/2023/09/generar-imagenes-responsive-picture.html

**Comment by TakoMics** [comment]
Muchas gracias por el tutorial Daniel, esto sirve muchísimo para estos nuevos tiempos donde los dispositivos móviles tienen ahora hasta más prioridad en las búsquedas que el clásico PC.

Mil gracias. Saludos.

**Reply by danieIabel** [reply]
Realmente espero que este tipo de artículos los cuales serán frecuentes, ayuden a desarrolladores a hacer mejores temas amigo. Muchas gracias por comentar

**Comment by danieIabel** [comment]
He actualizado el artículo agregando soporte para el servidor proxy de Google, el cual es el que se usa cuando la imagen no está subida a Blogger o se trata de un video de youtube. Gracias a este cambio en la función, se le pueden aplicar parámetros a estos enlaces.

**Comment by danieIabel** [comment]
He actualizado el funcionamiento de esta inclusion y ahora los servidores que permiten parámetros se detectan de manera más eficiente.
